examples/apps/lex-make-appointment/lambda_function.py

```diff
@@ -191,7 +192,7 @@ def lambda_handler(event, context):
                   args['arn'] + "- checking event details...")
             # Identify ARN in event detail
             for key, value in event['detail'].iteritems():
-                if isinstance(value, basestring):
+                if isinstance(value, string_types):
```

Comment:
Just a Python clarification question: `string_types` will cover both string and unicode in Python 2 right?

Reply:
Correct.  http://six.readthedocs.io/#six.string_types